android/src/main/java/com/reactnativecommunity/webview/RNCWebView.java

```diff
@@ -9,6 +10,7 @@
 
 public class RNCWebView extends FrameLayout {
   private static final String TAG = "RNCWebView";
+  public static final int INVALID_VIEW_ID = -1;
```

Comment:
https://developer.android.com/reference/android/view/View#findViewById(int)
>  if the ID is invalid (< 0) or there is no matching view in the hierarchy. 
I used -1 as it's a recognized invalid view id